charts/flink-job/values.yaml

```diff
@@ -200,6 +200,22 @@ flinkProperties:
   metrics.reporters: prom
   metrics.reporter.prom.class: org.apache.flink.metrics.prometheus.PrometheusReporter
 
+# -- (`map`) [Node selector] for Job Manager and Task Manager
+nodeSelector:
```

Comment:
Don't actually set values here. You are just putting in an empty key as a pointer for anyone reading the docs:

```yaml
# -- (`map`) [Node selector] for Job Manager and Task Manager
nodeSelector:

# -- (`map`) [Tolerations] for Job Manager and Task Manager
tolerations:
```

Reply:
Done